src/tcgwars/logic/impl/gen4/Platinum.groovy

```diff
@@ -3673,16 +3569,16 @@ public enum Platinum implements LogicCardInfo {
               onActivate {
                 eff = getter (GET_FULL_HP) {h->
                   def pcs = h.effect.target
-                  if (pcs.owner == self.owner && !pcs.name.contains("Shaymin")){
+                  if (pcs.owner == self.owner && !pcs.name.contains("Shaymin") && pcs.types.contains(G)) {
```

Comment:
I don't think `contains` is correct here. That would also affect an ex/EX/GX/V/VMAX as well.

Reply:
Changed it to do an exact string match